### DISCUSSION TITLE
724

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#908] [Comments]
# 724

기존 프로젝트에서 신규로 페이지를 만들었지만, 해당 페이지의 공유 링크는 기존 프로젝트의 meta tag가 아니라 신규 페이지 전용 meta tag 가 필요할 때가 종종 있다. 카톡방 공유 링크의 미리보기나 여러 크롤러들의 수집등에 유리함을 얻기 위해서는 Server Side에서 meta tag를 만들어서 HTML을 내려주는 것이 좋다. 하지만 아래와 같은 환경이라면 이렇게 Server Side를 동적으로 그려내기가 어려울 수 있다. 기존 프로젝트의 레거시로 인해서 동적으로 Server Side에서 meta tag를 수정하기가 어렵거나 SPA로 구축되어 단일 HTML로만 관리되고 있어 Server Side에서 meta tag 를 수정하기 어렵거나 그래서 원하는 meta tag를 가진 공유 링크만 담당하는..

https://jojoldu.tistory.com/724

**Comment by mystyle2006** [comment]
와~ 제가 전 직장에서 근무할 때 비슷한 상황이 있어 cloud front + lambda 트리거로 처리했던 경험이 있는데 그 당시 이렇게 해도 되나 했던 방식을 여기서 보니 매우 신기하네요!! :)

**Comment by dhksrl997** [comment]
저도 예전에 람다 사용할 때, 장애가 크게 발생했었는데 찾아보니
람다에 계정별로 동시성 제한이 있던데, 프로비저닝해서 사용하신건가요?
